tests/main.cpp

```diff
@@ -3,6 +3,14 @@
 
 #include "MangoAccount.hpp"
 
+TEST_CASE("Request Airdrop"){
+    const auto fromKey = solana::PublicKey::fromBase58(
+      "83astBRguLMdt2h5U1Tpdq5tjFoJ6noeGwaY3mDLVcri");
+      auto connection = solana::rpc::Connection();
+  auto result = connection.requestAirdrop(fromKey,1000000000,"http://localhost:8899");
```

Comment:
connection has it's own url defined, no need to override it when calling request Airdrop